feature-libs/user/profile/components/register/register-consents.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { UntypedFormArray, UntypedFormBuilder } from '@angular/forms';
+import { ConsentTemplate } from '@spartacus/core';
+
+@Injectable()
+export class RegisterConsentsService {
+  constructor(protected fb: UntypedFormBuilder) {}
+
+  loadAdditionalConsents(): {
+    template: ConsentTemplate;
+    required: boolean;
+  }[] {
+    return [];
+  }
+
+  generateAdditionalConsentsFormControl(): UntypedFormArray {
+    return this.fb.array([]);
+  }
+}
```

Comment:
I'm just curious why did we create a brand new component's service, maybe there was a reason... Did you consider instead adding this same this logic to the existing `RegisterComponentService` (and extending it in existing `CdcRegisterComponentService`)?

Reply:
I created a new service just to club all consent related methods to one place - i will merge it to already existing service if its a good practice

Reply:
No big need to move it. I was just curious about your reasoning.
I was thinking about moving it, because the `RegisterComponentService` is already a helper service for the component (e.g. loading the person's titles).

Reply:
okay .. any ways i moved it :)